include/download.mk

```diff
@@ -222,10 +223,10 @@ define DownloadMethod/rawgit
 	[ \! -d $(SUBDIR) ] && \
 	git clone $(OPTS) $(URL) $(SUBDIR) && \
 	(cd $(SUBDIR) && git checkout $(VERSION) && \
-	git submodule update --init --recursive) && \
+	$(if $(filter skip,$(SUBMODULES)),true,git submodule update --init --recursive -- $(SUBMODULES))) && \
 	echo "Packing checkout..." && \
 	export TAR_TIMESTAMP=`cd $(SUBDIR) && git log -1 --format='@%ct'` && \
-	rm -rf $(SUBDIR)/.git && \
+	find $(SUBDIR) -name .git | xargs rm -rf && \
```

Comment:
All submodule share .git on the root directory, so there is no need to modify.

Reply:
This is correct, however the submodules still have a `.git` file that points to the root WC / repo, which we are deleting. In my specific case leaving them in place tripped up a build script in the submodule. This is because the presence of the .git file makes it look like the tree is in a working copy, but it's broken as far as running any git commands goes (e.g. to extract the current commit hash).

Removing them avoids this issue (and also just seems cleaner overall than leaving these broken .git files in place).

Reply:
Actually this will unfortunately have the side-effect of changing the hash of tarballs with submodules where the submodule `.git` files weren't stripped... So to keep those stable I'll revert this line.